drivers/storage/portworx/portworx.go

```diff
@@ -685,7 +685,7 @@ func setDefaultAutopilotProviders(
 				Name: "default",
 				Type: "prometheus",
 				Params: map[string]string{
-					"url": "http://prometheus:9090",
+					"url": "http://px-prometheus:9090",
```

Comment:
Looks like we never had a unit test for this piece of code. Can we add a UT in this method - https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/portworx_test.go#L643?
Just check that there is a provider if you did not specify anything and it is equal to what we have here.